external-api/src/constants/emailTemplates.constant.ts

```diff
@@ -1,3 +1,3 @@
 export const EMAIL_TEMPLATES = {
-  ESTORE_FAILED: '50f4ba6c-9eef-4e90-bf86-5945cd08d0cb',
+  ESTORE_FAILED: '345ae136-7442-47b4-b26f-1a3201fead41',
```

Comment:
Just to confirm - do we need a ticket to remove the `50f4ba6c-9eef-4e90-bf86-5945cd08d0cb` template once deployed?

Reply:
I have already removed that template and updated with the latest one as the old template was not fit for purpose.